LondonBritishLibrary/orient/BLorient615.xml

```diff
@@ -0,0 +1,351 @@
+        <extent>
+            <measure unit="leaf">185</measure>
+            <dimensions type="outer" unit="in">
+                <height>11.25</height>
+                <width>10.125</width>
+            </dimensions>
+        </extent>
+    </supportDesc>
+
+    <layoutDesc>
+        <layout columns="2" writtenLines="18" cert="high">
+        </layout>
+    </layoutDesc>
+
+  </objectDesc>
+
+  <handDesc>
+  <handNote script="Ethiopic" xml:id="h1">
+  <desc>Written in a fine hand, of the beginning of the <date>18th century.</date></desc>
+  <date notBefore="1700" notAfter="1730" resp="PRS10171Wright"/>
+  </handNote>
+  </handDesc>
+
+<decoDesc>
+
+  <summary>The miniatures are painted in the <term key="SecondGondarineStyle">Second Gondarine style.</term></summary>
+
+<decoNote type="miniature" xml:id="d1">
+<locus target="#8r"/>
+<desc><term key="angel">Archangel</term> in triumph over Satan.</desc>
+</decoNote>
+
+<decoNote type="miniature" xml:id="d2">
+<locus target="#8v"/>
+<desc><term key="angel">Archangel</term> with a <term key="sword">sword</term> overseeing sinners in hell.</desc>
+</decoNote>
+
+<decoNote type="miniature" xml:id="d3">
+<locus target="#103v"/>
+<desc>Annunciation with a female <term key="donordepiction">donor</term> prostrating to Virgin Mary.</desc>
+</decoNote>
+
+<decoNote type="miniature" xml:id="d4">
+<locus target="#134v"/>
+<desc><term key="angel">Archangel</term> with a female <term key="donordepiction">donor</term> in prostration.</desc>
+</decoNote>
+
+<decoNote type="miniature" xml:id="d5">
+<locus target="#182v"/>
+<desc><term key="angel">Archangel</term> killing a sea monster.</desc>
+</decoNote>
+
+<decoNote type="miniature" xml:id="d6">
+<locus target="#183r"/>
+<desc><term key="angel">Archangel</term> cures a man's blindness.</desc>
+</decoNote>
+
+<decoNote type="miniature" xml:id="d7">
+<locus target="#183v"/>
+<desc>In the upper register <term key="angel">archangel</term> saving people at sea, in the lower - healing a man possessed by a <term key="demon">demon</term>.</desc>
+</decoNote>
+
+
+</decoDesc>
+
+  <additions>
+
+  <list>
+    <item xml:id="a1">
+    <locus target="#2r"></locus>
+      <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+              <placeName ref="INS0101MadhaneAlam"/>.</desc>
+          <q xml:lang="gez">ድርሳነ፡ ሚካኤል፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+</item>
+
+<item xml:id="a2">
+<locus target="179v"/>
+<desc type="MixedNote">List of drugs</desc>
+<q xml:lang="am"><seg part="I">ኈልቈ፡ መድኃኒት፤ ዕፀ፡ ምናሄ፤ ዕፀ፡ ዘዌ፤ አሜራ፤ ሰመግ፤ ፅፍሬና፤ ዘርጫ፡ የምድር፡ እምባይ፤ በቶ፤ አሰረት፤ የምድር፡ ቁግ፤ የመካን፡ እንዶድ፡ ስር፤</seg></q>
+</item>
+
+<item xml:id="a3">
+<locus target="#111v"/>
+<desc type="Supplication">Name of the first owner erased; earliest substitution <foreign xml:lang="gez"><persName role="owner" ref="PRS13461Anagda">አናግደ፡
+ወለተ፡ ሚካኤል፡</persName></foreign> e. g. on <locus target="#111v">f. 111v</locus>, but also erased.</desc>
```

Comment:
I think it's really better not to write insice `<locus>` in such cases, so as not increasing already existing f./fol. inconsistencies (also above)

Reply:
I hope it's better now.